package.json

```diff
@@ -9,10 +9,11 @@
   ],
   "private": true,
   "scripts": {
-    "build": "tsc -p tsconfig.app.json"
+    "build": "tsc -p tsconfig.app.json",
+    "test": "tsc -NoEmit"
   },
   "dependencies": {
-    "renovate": "25.21.13",
+    "renovate": ">=24",
```

Comment:
This is the major caching optimizer, so it will not change and the lockfile will only change on other updates, so hopefully less digest updates because of this changes